lib/external.ts

```diff
@@ -0,0 +1,45 @@
+import * as Promise from 'bluebird';
+import { DockerProgress } from 'docker-progress';
+import * as Dockerode from 'dockerode';
+import * as _ from 'lodash';
+
+import { BuildTask } from './build-task';
+import { BuildProcessError } from './errors';
+import { LocalImage } from './local-image';
+
+const hasImageTag = (name: string): boolean => {
+	const tagRegex = /.+:[^/]+$/;
+	return tagRegex.test(name);
```

Comment:
This will match a digest which is slightly confusing given the function name.

Reply:
In docker terms, a digest is also an image tag, this is fine IMO - I can change the naming if you like though?